tasklib/task.py

```diff
@@ -111,23 +111,11 @@ class TaskResource(SerializingObject):
     read_only_fields = []
 
     def _load_data(self, data):
-        self._data = data
+        self._data = dict((key, self._deserialize(key, value))
+                          for key, value in data.items())
         # We need to use a copy for original data, so that changes
-        # are not propagated. Shallow copy is alright, since data dict uses only
-        # primitive data types
-        self._original_data = data.copy()
-
-    def _update_data(self, data, update_original=False):
```

Comment:
I realized there's actually a problem with removing ```only_fields```. With that functionality being removed, consider the following scenario:

```
>>> task = tw.tasks.get(id=4)
>>> task['project'] = 'Home'
>>> task.annotate('I should really do this task')
>>> task.save()
```

Task would get annotated, but since we refresh all the fields at that point, any unsaved modification will perish.

```
>>> task['project'] == None
True
```

So I guess maybe this was the forgotten reason? In either way, we should keep it.